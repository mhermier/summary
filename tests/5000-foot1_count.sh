#!/bin/sh

utils/generate-test-input.sh 0 100 0 | ./summary --foot 1 --count --count-skipped --count-matched
