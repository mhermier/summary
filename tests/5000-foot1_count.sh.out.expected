... 99 entries skipped
entry-100
1/100 entries matched
